plugins/notifications-frontend/src/components/SendNotification/SendNotification.tsx

```diff
@@ -0,0 +1,207 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+
+import { makeStyles } from '@material-ui/core';
+import Alert from '@mui/material/Alert';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+
+import { notificationsApiRef, NotificationsCreateRequest } from '../../api';
+
+const useStyles = makeStyles({
+  container: {
+    width: '60%',
+    minWidth: '30rem',
+  },
+});
+
+export const SendNotification = () => {
+  const notificationsApi = useApi(notificationsApiRef);
+  const configApi = useApi(configApiRef);
+  const styles = useStyles();
+
+  const [notificationId, setNotificationId] = React.useState<string>();
+  const [error, setError] = React.useState<string>();
+
+  const [origin, setOrigin] = React.useState<string>('my-origin');
+  const [title, setTitle] = React.useState<string>('my-title');
+  const [message, setMessage] = React.useState<string>();
+  const [topic, setTopic] = React.useState<string>();
+  const [targetUsers, setTargetUsers] = React.useState<string[]>();
+  const [targetGroups, setTargetGroups] = React.useState<string[]>();
+  const [actions, setActions] = React.useState<string>();
+
+  const handleSubmit = async () => {
+    try {
+      const parsedActions = actions ? JSON.parse(actions) : undefined;
+
+      const notification: NotificationsCreateRequest = {
+        origin,
+        title,
+        message,
+        actions: parsedActions,
+        topic,
+        targetUsers,
+        targetGroups,
+      };
+
+      const id = await notificationsApi.createNotification(notification);
+      setNotificationId(id);
+    } catch (_e) {
+      const e = _e as Error;
+      setError(e.message);
+    }
+  };
+
+  const getCurl = () => {
+    const data: NotificationsCreateRequest = {
+      title,
+      origin,
+    };
+
+    try {
+      if (message) {
+        data.message = message;
+      }
+      if (topic) {
+        data.topic = topic;
+      }
+      if (actions) {
+        data.actions = JSON.parse(actions);
+      }
+      if (targetUsers && targetUsers.length > 0) {
+        data.targetUsers = targetUsers;
+      }
+      if (targetGroups && targetGroups.length > 0) {
+        data.targetGroups = targetGroups;
+      }
+
+      return `curl -X POST ${configApi.getString(
+        'backend.baseUrl',
+      )}/api/notifications/notifications -H "Content-Type: application/json" -d '${JSON.stringify(
```

Comment:
This should really use `discoveryApi` to get the API path

Reply:
This whole SendNotifications screen is questionable. I still keep it there, mostly for demo or development. But IMO, there is no end-user use-case for it. I tend to removing it.

It is expected that most of the messages will be created by external systems or other BE plugins.
FE plugins can create them as well but the main NotificationsPage has recently no use-case for it.